binrz/man/rz-gg.1

```diff
@@ -1,13 +1,14 @@
-.Dd Sep 30, 2014
+.Dd Jan 24, 2024
 .Dt RZ_GG 1
 .Sh NAME
 .Nm rz-gg
 .Nd rizin frontend for r_egg, compile programs into tiny binaries for x86-32/64 and arm.
```

Comment:
`RzEgg` is better than `r_egg`, also not only x86 and ARM, it supports more architectures, just say "different architectures".